app/react/kubernetes/namespaces/components/ResourceQuotaFormSection/ResourceQuotaFormSection.tsx

```diff
@@ -57,13 +59,34 @@ export function ResourceQuotaFormSection({
           <div className="flex flex-row">
             <FormSectionTitle>Resource Limits</FormSectionTitle>
           </div>
+
+          {(!cpuLimit || !memoryLimit) && (
+            <div>
```

Comment:
Does the FormError or TextTip have the same styles?

Reply:
Yes